packages/builder/src/components/userInterface/PropertyControl.svelte

```diff
@@ -17,21 +62,35 @@
         innerVal = props.valueKey ? v.target[props.valueKey] : v.target.value
       }
     }
-    onChange(key, innerVal)
+    replaceBindings(innerVal)
   }
 
   const safeValue = () => {
+    getBindableProperties()
+    let temp = value
+    const boundValues = (value && value.match(/{{([^}]+)}}/g)) || []
```

Comment:
NAB: possibly worth moving these regex definitions out into a constant so it's easy to see what it's matching for at a glance

```
const CAPTURE_VAR_INSIDE_MUSTACHE = /{{([^}]+)}}/g
```